src/builder/create_application_command.rs

```diff
@@ -129,6 +130,21 @@ impl CreateApplicationCommandOption {
 
         self
     }
+
+    /// If an option is a Channel type, it will only be able to show these types.
+    pub fn add_channel_types(&mut self, channel_types: Vec<ChannelType>) -> &mut Self {
```

Comment:
Name should be just `channel_types`, because we have established that `add_*` methods on builders will append data, not overwrite. The option type `Channel` should also be referenced, just like in the above method's documentation.
```suggestion
    /// If the option is a [`Channel`], it will only be able to show these types.
    ///
    /// [`Channel`]: crate::model::interactions::application_command::ApplicationCommandOptionType::Channel
    pub fn channel_types(&mut self, channel_types: Vec<ChannelType>) -> &mut Self {
```